spark/v3.2/spark/src/jmh/java/org/apache/iceberg/spark/source/IcebergSourceFlatDataBenchmark.java

```diff
@@ -54,6 +54,8 @@ protected final Table initTable() {
     HadoopTables tables = new HadoopTables(hadoopConf());
     Map<String, String> properties = Maps.newHashMap();
     properties.put(TableProperties.METADATA_COMPRESSION, "gzip");
+    // Force usage of zstd compression for all Iceberg Source Flat Data Benchmarks that use Parquet
+    properties.put(TableProperties.PARQUET_COMPRESSION, "zstd");
```

Comment:
Can you remove the unnecessary changes like this? If you want to do this, then let's move it to a separate PR.